### PR TITLE
rg-180: Fix migration add id generation

### PR DESCRIPTION
**Pull request by siefimov**

### CODE REVIEW

File: backend/src/seeds/seed-types/seed-data.type.ts [hunk 1]
```diff
@@ -26,6 +30,8 @@ type Resume = {
     created_at: Date;
     updated_at: Date;
     deleted_at: Date;
+    user_id: string;
+    template_id: string;
```
Comment (honkster-nazarii) [inline backend/src/seeds/seed-types/seed-data.type.ts:34]: why does profile need template_id

**Review by honkster-nazarii — CHANGES_REQUESTED** [review]

File: backend/src/seeds/seed-types/seed-data.type.ts (hunk 1 above)
Comment (siefimov) [reply backend/src/seeds/seed-types/seed-data.type.ts:34]: it is for resume, not profile. 
please expand hidden rows

**Review by honkster-nazarii — APPROVED** [review]

**Review by MaxPoplavskiy — APPROVED** [review]

**Review by Icerock10 — APPROVED** [review]
It works) I've tested

**Review by Valentyn13 — APPROVED** [review]

**Review by jurok3x — CHANGES_REQUESTED** [review]
image

**Comment by Icerock10** [comment]
It seems like after each migration change not related to the current - it throws errors

File: backend/src/migrations/20240304112515_add_id_generation_to_database.ts
Comment (Vitaliy-Kalenichenko) [inline backend/src/migrations/20240304112515_add_id_generation_to_database.ts:1]: Don't edit migrations

Comment (siefimov) [reply backend/src/migrations/20240304112515_add_id_generation_to_database.ts:1]: Ok.

**Comment by siefimov** [comment]
> image

Most likely, this error cannot be related to this migration, because it hasn't merge to dev yet.

**Review by Valentyn13 — APPROVED** [review]
I merge it with branch which was based on latest dev and migrations run without errors. Please anyone check in your local environment too

**Review by MaxPoplavskiy — APPROVED** [review]

File: backend/src/migrations/20240304112515_add_id_generation_to_database.ts
Comment (siefimov) [reply backend/src/migrations/20240304112515_add_id_generation_to_database.ts:1]: I reverted change.

**Review by RevolveStrm — APPROVED** [review]

**Comment by serhiy-yefimov** [comment]
[image.png](https://trello.com/c/WQcUzuS0/180-imagepng)

**Review by jurok3x — APPROVED** [review]

File: backend/src/migrations/20240304112515_add_id_generation_to_database.ts
Comment (MaxPoplavskiy) [reply backend/src/migrations/20240304112515_add_id_generation_to_database.ts:1]: Changes are still here😔

**Review by MaxPoplavskiy — APPROVED** [review]

**Review by honkster-nazarii — APPROVED** [review]

**Review by RevolveStrm — APPROVED** [review]

**Pull request merged**